modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -14579,6 +14579,10 @@ set-default-shipping-address=Set Default Shipping Address
 set-element-as-click-target=Set Element as Click Target
 set-expiration-date=Set Expiration Date
 set-filters=Set Filters
+set-lar-cleanup-job-interval=LAR Cleanup Job Execution Frequency
```

Comment:
If rename to `cleanup-job-interval` then we can update this text as `Cleanup Job Execution Frequency`